### DISCUSSION TITLE
Ability to send diamonds directly to a user.

### DISCUSSION THREAD

**Discussion by pangia** [deso-protocol/dips#177] [Medium CIPs]
Ability to send diamonds directly to a user without the need for an associated post. This is needed for games that use diamonds as the base currency.

**Comment by ItsAditya-xyz** [comment]
I support this!
I don't know about the technical possibility of this but this feature will be helpful in many different projects.
We built DeSo-Discord integration through which you can tip users directly in discord. Currently it's BASIC_TRANSFER but it would be great if we can switch to giving diamond to profile infinitely.

**Reply by marlon-montgomery** [reply]
+ 1 as we are looking to do this with mousai.stream and currently need to go the path like @aditya did with Cordify or create posts for the artists and send all diamonds to it. A drawback of creating posts method, is that bitclout.com allows users to hide posts, which prevents any further transactions on the post.

**Comment by oO** [comment]
I agree. Yes, you could just send $deso from one account to another, but what's important is for the transaction and the notification to appear as a ( single, double, 3x, ... ) diamond, to maintain the same concepts for users. ( Diamonds = Tips, $deso = payments )

**Comment by stet** [comment]
One hacky implementation would be to have a post that you designate as a bio extender and this post's content can be displayed as appended content to an exisiting bio content. 

But the problem is that you cannot diamond a level to a post more than once. 
So tipping a profile would be limited to some extent. 
For example if I tip you Level5 and want to tip you Level5 again few weeks later, I cannot. 
Unless I am mistaken, the API does not let you get around this (I thought it did at one point). 
Correct me if I am wrong.

Another approach would be to add a memo text to a BasicTransfer (sending deso directly to a public address).
The memo could be "tip" or "diamond" and then this data could be used to track tips received as opposed to other kinds of "payments". UI could provide a dropdown of labels to use in a memo field if not free-form.

**Reply by darian-parrish** [reply]
I like the idea of a memo field for transactions but mainly for other use cases. If that is the easiest way to implement, that could work here but definitely pushes more of the work onto the UI/UX designers.

**Comment by sungkhum** [comment]
This would be a great way for 3rd party sites to easily implement "tipping" - great idea.

**Reply by darian-parrish** [reply]
💯
